spec/controllers/your_details_controller_spec.rb

```diff
@@ -100,6 +102,63 @@ module Wpcc
       end
     end
 
+    describe 'sessions' do
+      after { Timecop.return }
+
+      it "'wpcc_expires_at' key should always be updated" do
+        session[:wpcc_expires_at] = 5.minutes.ago
+        get :new, nil, some_session
+
+        expect(session[:wpcc_expires_at]).to be > 5.minutes.ago
+      end
+
+      context 'when session has expired' do
+        before do
+          wpcc_expires_at = 5.minutes.ago
+
+          Timecop.travel(Time.current + 30.minutes)
```

Comment:
Can we move this line? I found it confusing that this is between setting the `wpcc_expires_at` variable and the `get`
This perhaps should be at the top of the actual test, considering there's only one test in this context.